### ISSUE TITLE
Check packages sizes before npm publish and fail if the package is bigger than expected

### ISSUE THREAD

**Issue by Jolg42** [prisma/prisma#10356]
## Problem
I just found randomly that the size of the `@prisma/sdk` package jumped from less than 1MB to 120MB after our internal build/bundling refactoring so fixed it with https://github.com/prisma/prisma/pull/10354

## Solutions

https://github.com/pkg-size/action
https://github.com/bundlewatch/bundlewatch
https://remarkablemark.org/blog/2020/12/20/size-limit-github-actions-workflow/
https://github.com/mcataford/packwatch 

Basically running a `pnpm pack --dry-run` and analyzing the size might do the job in our publishing script.

**Comment by jkomyno** [comment]
Hi there,
As part of our effort to clean up our GitHub repo, we’re closing this issue to help focus on features and improvements that align with our focus areas and those of our community. 

This issue was originally created for internal tracking purposes, but if you feel this is important to you, feel free to reopen the issue and get like-minded users to upvote as we will prioritize fixing issues based on community interest.

For more insight into our priorities and how we’re thinking about the future of Prisma, please see our blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and support!
